airflow/config_templates/default_airflow.cfg

```diff
@@ -591,6 +591,11 @@ json_format = False
 # Log fields to also attach to the json output, if enabled
 json_fields = asctime, filename, lineno, levelname, message
 
+[elasticsearch_configs]
+
+use_ssl = False
+verify_certs = True
+
```

Comment:
**ca_certs** is missing ??

Reply:
These are just example configs. Users can add anything in the elasticsearch SDK